docs.md

```diff
@@ -0,0 +1,239 @@
+</p>
+
+## - m2nTime(time)
+`time: (string) => Time to convert in {h:mm:ss} format`<br>
+`return: (string) => Returns equivalent time in normal clock system`<br><br>
+This functions converts given metric system time to equivalent normal clock system time. The input format must be h:mm:ss.<br />
+Following code shows an example on how to use it
+```js
+export default function App() {
+	return (
+		<View style={styles.container}>
+		<Text>Metric('5:50:00') = Normal ('{clock.m2nTime('5:50:00')}')</Text> 			
+		</View>
+	);
+}
+```
+
+Which results in 
+<p align="center">
+  <img src="https://user-images.githubusercontent.com/47807051/143627652-48ce66d0-4b39-48f9-b290-39a52ed3428e.png" alt="m2nTime"/>
+</p>
+
+*You get the gist, right? You can figure out the other functions...*<br />
+*What?? You can't! BOO-HOO-HOO I'm a developer still I don't know how to figure small things out.*<br />
+*Did I hurt your petty little feeligns there buddy? Oh my bad...But sorry this ain't google which just shout out billion ways to centre a div.*<br />
+
+## - m2nDate(date)
+`date: (string) => Date to convert in {dd-m-yyyy} format`<br>
+`return: (string) => Returns equivalent date in normal clock system`<br><br>
+*Sorry fellas, I'm bored, don't wanna describe, you know how it'll work, here's example and output:*
+```js
+export default function App() {
+	return (
+		<View style={styles.container}>
+		<Text>Metric('54-2-0730') = Normal ('{clock.m2nDate('54-2-0730')}')</Text>	
+		</View>
+	);
+}
+```
+
+Which results in 
+<p align="center">
+  <img src="https://user-images.githubusercontent.com/47807051/143637477-9c6e7a77-6aec-4496-8de1-5ee4101a730a.png" alt="m2nDate"/>
+</p>
+
+## - n2mTime(time)
+`time: (string) => Time to convert in {hh:mm:ss} format`<br>
+`return: (string) => Returns equivalent time in metric clock system`<br><br>
+Following code shows an example on how to use it
+```js
+export default function App() {
+	return (
+		<View style={styles.container}>
+		<Text>Normal ('13:12:00') = Metric ('{clock.n2mTime('13:12:00')}')</Text> 			
+		</View>
+	);
+}
+```
+Which results in 
+<p align="center">
+  <img src="https://user-images.githubusercontent.com/47807051/143639190-b688ff1b-cfbc-4e98-8126-80958d517f8a.png" alt="n2mTime"/>
+</p>
+
+## - n2mDate(date)
+`date: (string) => Date to convert in {dd-mm-yyyy} format`<br>
+`return: (string) => Returns equivalent date in metric clock system`<br><br>
+Following code shows an example on how to use it
+```js
+export default function App() {
+	return (
+		<View style={styles.container}>
+		<Text>Normal ('30-11-1999') = Metric ('{clock.n2mDate('30-11-1999')}')</Text> 			
+		</View>
+	);
+}
+```
+
+Which results in 
+<p align="center">
+  <img src="https://user-images.githubusercontent.com/47807051/143640604-2953de75-a30e-48fc-aa8f-8a8b0f6b8bad.png" alt="n2mDate"/>
+</p>
+
+*For anyone wondering why I chose 30-nov and not 14-May! Well that's none of your business...but 30th nov is my birthday*
```

Comment:
"that's none of your business" please -> professionalism, I got feeling you are attacking me and now I am PISSED OFF like John Wick